feature/gribi/otg_tests/basic_encap_test/metadata.textproto

```diff
@@ -22,7 +22,6 @@ platform_exceptions:  {
     vendor: JUNIPER
   }
   deviations:  {
-    ttl_copy_unsupported: true
```

Comment:
Should we also remove this from `metadata.proto` and also https://github.com/openconfig/featureprofiles/blob/ddc6255868db1fbb7eb917e48771800f119d0963/internal/deviations/deviations.go#L739 ?

Reply:
I was thinking about separate CL to clean up the code. This being clean up the metadata.

Reply:
SGTM.